julian-gaviria-1121917910/tarea_6/tarea-6.js

```diff
@@ -0,0 +1,132 @@
+//tarea numero 6 Repaso manipulación de objetos y arrays
+// coded By Juliangadev
+/* 
+6.1. Ejercicio de Bucle For con Arrays:
+Dado el siguiente array de números:
+
+let numeros = [1, 2, 3, 4, 5];
+Utiliza un bucle for para multiplicar cada número por 2 y almacenar el resultado en un nuevo array. Luego, imprime el nuevo array.
+*/
+//array numeros
+let numeros = [1, 2, 3, 4, 5];
+//Utiliza un bucle for para multiplicar cada número por 2 y almacenar el resultado en un nuevo array. Luego, imprime el nuevo array.
+let numeros2 =[]
+for (let i = 0; i < numeros.length; i++) {
+   numeros2.push(numeros[i] * 2 )
+   
+}
+console.log("6.1. Ejercicio de Bucle For con Arrays")
+console.log(numeros2)
+
+/* 
+6.2 Ejercicio de Búsqueda en Arrays de Objetos:
+Dado el siguiente array de objetos que representan productos:
+let productos = [
+  { nombre: "Camisa", precio: 20 },
+  { nombre: "Pantalón", precio: 30 },
+  { nombre: "Zapatos", precio: 50 },
+];
+Utiliza un bucle for para encontrar el producto con el precio más bajo. Luego, imprime el nombre y el precio de ese producto.
+*/
+// let productos = [
+//     { nombre: "Camisa", precio: 20 },
+//     { nombre: "Pantalón", precio: 30 },
+//     { nombre: "Zapatos", precio: 50 },
+//   ];
+// console.log("6.2 Ejercicio de Búsqueda en Arrays de Objetos")
+//  let productPrice = []
+//  let lowest
+//   for (let i = 0; i < productos.length; i++) {
+//     productPrice.push(productos[i].precio)
+//     lowest = Number(Math.min(...productPrice))
+//     if (productos[i].precio === lowest) {
+//         console.log(`el producto mas barato es ${productos[i].nombre} con un precio de $${productos[i].precio} pesos`)
+//     }
+//   }
+//   console.log(productPrice)//imprime el array de los precios
+//   console.log(lowest)//imprime el precio mas barato
+let productos = [
+  { nombre: "Camisa", precio: 20 },
+  { nombre: "Pantalón", precio: 30 },
+  { nombre: "Zapatos", precio: 50 },
+];
+console.log("6.2 Ejercicio de Búsqueda en Arrays de Objetos")
+
+let lowest = 20
```

Comment:
que pasa si yo cambio el precio de la Camisa de 20 a 70 seguiria funcionando este codigo?

Reply:
profe pero los otros compañeros harcodearon el precio tambien y lo veo que ud les "mergio" (?) no entiendo que tenia mal en el primer pull
Por ejemplo:
![image](https://github.com/sergio-ceballos-intelex/tareas-modulo-3-grupo-8/assets/86694455/d3e406cf-1a82-4c7c-bfce-b9a0d563b594)

Reply:
pero mira la diferencia, ahi se esta tomtando productos en la primera posicion, no estan quemando directamente el valor 20...ademas si el segundo producto tiene un precio menor a 20 no funcionaria ya que estas comparando con === con lo cual no entrara al if...tambien el console se debe hacer afuera del for ya que puede que muchos productos salgan en el console ya que se puede dar que haya uno barato pero despues haya otro mas barato y el console saldria para los dos

Reply:
entiendo